src/test/connection_snippets.rs

```diff
@@ -0,0 +1,20 @@
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+
+use crate::error::Result;
+use crate as mongodb;
```

Comment:
hacky aliasing so that the example uses `mongodb` instead of `crate`

Reply:
Earlier in the driver's development, we moved all of the external tests from the `tests` directory to the `test` submodule so that we could share code between our internal and external tests. However, since we're writing this test as a way to document usage of the driver, we know we won't ever be using any internal or test-only features of the driver, so this might be a good candidate to put in the `tests` directory at the top-level of the repo, in which case we wouldn't need to do the aliasing.

Reply:
makes sense, I'll move it